app/src/main/java/com/li/weatherapp/base/BaseViewHolder.kt

```diff
@@ -0,0 +1,9 @@
+package com.li.weatherapp.base
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+
+abstract class BaseViewHolder<T>(private val itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    abstract fun onBind(item: T)
```

Comment:
```suggestion
    abstract fun bindData(item: T)
```
chú ý cách đặt tên hàm. Các hàm có tiền tố `on` thường để xử lý các sự kiện callback